firmware/spade/src/rpi/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ target_compile_definitions(spade PRIVATE
   # PICO_DEFAULT_UART=0
   # PICO_DEFAULT_UART_TX_PIN=28
   # PICO_DEFAULT_UART_RX_PIN=29
+  JERRY_SYSTEM_ALLOCATOR=1
```

Comment:
why are you switching to the system allocator here? (instead of the default jerryscript BSS block)